dali/pipeline/operators/op_schema.h

```diff
@@ -152,6 +152,15 @@ class DLL_PUBLIC OpSchema {
     return *this;
   }
 
+  /**
+   * @brief Notifies that grouping this op with Logical Id to initialize the group with common
+   * seed and support Multiple Input Sets is not supported
+   */
+  DLL_PUBLIC inline OpSchema& DisallowSyncSeeds() {
```

Comment:
I'd say that `DisallowInstanceGrouping` is a more descriptive name for this concept.

Reply:
Done